src/ci/docker/run.sh

```diff
@@ -82,6 +82,7 @@ exec docker \
   --env TRAVIS_BRANCH \
   --volume "$HOME/.cargo:/cargo" \
   --volume "$HOME/rustsrc:$HOME/rustsrc" \
+  --privileged \
```

Comment:
Maybe add a comment, why this is needed? It's in the commit message, but it would be more visible here.